src/launch/lib/service/template/functions.py

```diff
@@ -158,7 +158,7 @@ def render_jinja_template(
     if not template_data.get("data"):
         template_data["data"] = {}
 
-    env = Environment(loader=FileSystemLoader(template_path.parent))
+    env = Environment(loader=FileSystemLoader(template_path.parent), autoescape=False)
```

Comment:
This is what the CodeQL report is asking for:

```suggestion
    env = Environment(loader=FileSystemLoader(template_path.parent), autoescape=True)
```